recipe/meta.yaml

```diff
@@ -12,7 +12,26 @@ source:
 
 
 build:
-  number: 4
+  number: 5
+
+  # add build string so packages can depend on
+  # mpi or nompi variants explicitly:
+  # `dagmc * mpi_mpich_*` for mpich
+  # `dagmc * mpi_openmpi_*` for ßopenmpi
+  # `dagmc * nompi_*` for no mpi
+
+  {% if mpi != 'nompi' %}
+  {% set mpi_prefix = "mpi_" + mpi %}
```

Comment:
Is the `mpi_` part redundant?  I think I've seen other examples where the options are
* `mpich_`
* `openmpi_`
* `nompi_`
Perhaps the `mpi_` prefix is useful to make it easier to choose one without caring which???

Reply:
This seems to be the standard method to setting the variant in other packages, so I just copy and pasted that.